components/DevWorkshopAd.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { Alert } from 'reactstrap'
+import moment from 'moment'
+
+export default () => {
+  // Hide ad once the event starts
+  if (moment().isAfter('2018-04-24 19:00:00.000-05')) return null
+
+  return (
+    <Alert color="primary">
+      <h6>Want to work on the queue?</h6>
+      <p className="mb-0">
+        Join us for a development workshop on Thursday, April 24th at 7PM in
```

Comment:
Tuesday?

Reply:
time is a social construct

(will fix 😛)